### ISSUE TITLE
Support unicode in starters preview images

### ISSUE THREAD

**Issue by robinmetral** [gatsbyjs/gatsby#11901]
I'm not sure what we're using to generate the starter preview images, but it doesn't support unicode characters.
(Also the generated image on my monitor is 700x525px, scaled to 1080x810px. Blurry.)

:arrow_right: [Example with the Calpa Blog Starter](https://www.gatsbyjs.org/starters/calpa/gatsby-starter-calpa-blog/)

![screenshot_2019-02-19 gatsby-starter-calpa-blog](https://user-images.githubusercontent.com/35560568/53036902-1fc1ea00-3479-11e9-8de9-63aa7cebbeee.png)

**Comment by wardpeet** [comment]
We're currently using https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-transformer-screenshot which loads into a lambda. This is a stripped down Linux version that might cause the error.

**Comment by robinmetral** [comment]
So this tool (loving it by the way :heart_eyes:) uses [puppeteer](https://github.com/GoogleChrome/puppeteer/).

As suggesting in [this comment on an issue](https://github.com/GoogleChrome/puppeteer/issues/3668#issuecomment-456907132), maybe upgrading to 1.11.0 would do the trick? ([currently puppeteer is at 0.10.2](https://github.com/gatsbyjs/gatsby/blob/d33b3ec1185feef75e1242962e915eff07b9b43a/packages/gatsby-transformer-screenshot/lambda/package.json#L3))

I can try to run this locally and submit a PR, but would it work on my end without the S3 credentials?

**Comment by robinmetral** [comment]
Thanks gatsbot, but I don't think the issue is stale!

This is a possible solution:

> As suggesting in [this comment on an issue](https://github.com/GoogleChrome/puppeteer/issues/3668#issuecomment-456907132), maybe upgrading to 1.11.0 would do the trick? ([currently puppeteer is at 0.10.2](https://github.com/gatsbyjs/gatsby/blob/d33b3ec1185feef75e1242962e915eff07b9b43a/packages/gatsby-transformer-screenshot/lambda/package.json#L3))

**Comment by robinmetral** [comment]
I tried to upgrade `pupetteer` and to test it locally, but it didn't fix it.

Here's what I did:

1. [Set up my local environment](https://www.gatsbyjs.org/contributing/setting-up-your-local-dev-environment/)
2. Updated `pupetteer` to v.1.11.0 in [`packages/gatsby-transformer-screenshot/lambda/package.json`](https://github.com/gatsbyjs/gatsby/blob/d33b3ec1185feef75e1242962e915eff07b9b43a/packages/gatsby-transformer-screenshot/lambda/package.json#L3)
3. Ran `yarn` in the `lambda` directory to upgrade the package (seemed to work but I can't see `pupetteer` in the root's `yarn.lock`, am I doing it wrong?)
4. Spin up an instance of the website (90' build time :open_mouth: )

Does that mean upgrading `pupetteer` isn't the solution? Or am I missing a step? (I'm also noticing that some of the starter library features are missing in my instance)
